src/main/java/core/basesyntax/db/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+public class FruitTransaction {
+    private String fruit;
+    private int quantity;
+    private String type;
+
+    public FruitTransaction(String fruit, int quantity) {
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
```

Comment:
The class fields `type`, `fruit`, and `quantity` should be private to follow the encapsulation principle. Please add the appropriate access modifier.